content/integrations/azure.md

```diff
@@ -18,25 +18,73 @@ sidebar:
 
 ### Overview
 
-Datadog's Azure integration currently monitors Virtual Machines (diagnostics must be enabled), and SQL Databases. For any other service, please reach out to [support@datadoghq.com][3].
+Datadog's Azure integration currently monitors Virtual Machines, SQL Databases, and App Services. For any other service, please reach out to [support@datadoghq.com][3].
 Datadog's agent can also be deployed on your Azure virtual machines.
```

Comment:
@irabinovitch Definitely at some point. With WPC in two days I won't have the time to create them before hard.